pipelines/build-test-tool-template.yaml

```diff
@@ -17,7 +17,24 @@ steps:
     inputs:
       arguments: '-c $(BuildConfiguration)'
 
-  - task: DotNetCoreCLI@2
-    displayName: Run tests
+  - pwsh: |
```

Comment:
As we saw yesterday if you are using the `DotNetCoreCLI@2` task you won't need the manual upload task.

Reply:
@Evangelink We won't be able to use the upload test results functionality from DotNetCoreCLI anyways I think (at least for now).

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/dotnet-core-cli-v2?view=azure-pipelines

> This option appends `--logger trx --results-directory $(Agent.TempDirectory)` to the command line arguments.

Reply:
I think if we go with DotNetCoreCLI, it will look like the following:

```yml
  - task: DotNetCoreCLI@2
    displayName: Run unit tests
    inputs:
      command: 'test'
      publishTestResults: false
      nobuild: true
      configuration: '$(BuildConfiguration)'
      arguments: '-- --report-trx --results-directory $(Build.ArtifactStagingDirectory)/artifacts/TestResults --coverage --coverage-output $(Build.ArtifactStagingDirectory)/artifacts/Coverage/coverage.cobertura.xml --coverage-output-format cobertura'
```

I'll make the change once when I get it green.